### ISSUE TITLE
Lexicon - Header search button sizing

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1574]
**Describe the bug**
The search magnifying glass button found in the main search in the header of the control centre is currently 32px x 21px in sizing. Target spacing exemptions are not met due to the button overlaying the input.

**To Reproduce**
1. Go to Pulsar Lexicon
2. See sizing of Search submit button

**Expected behavior**
The size of all interactive elements plus the spacing between them should be at least 24 x 24 pixels.

**Screenshots**
<img width="494" alt="Image" src="https://github.com/user-attachments/assets/ac4a74a0-2222-4098-b320-c0a5ff53fe0a" />

**Additional context**
Related success criteria:
[2.5.8 Target Size (Minimum) (AA)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)

**Suggested fix**
Increase the size of the <button> element to at least 24 px in height
